websites/N/Navidrome/presence.ts

```diff
@@ -0,0 +1,42 @@
+const presence = new Presence({
+	clientId: "1126923786600583339",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+		largeImageKey: "https://i.imgur.com/mhD2eIr.png",
+	};
+
+	// Determine player state
+	let playerState;
+	// Check if audio-title has any content
+	if (document.querySelectorAll(".audio-title")[0].innerHTML !== "") {
+		if (document.querySelectorAll(".react-jinke-music-player-pause-icon")[0])
```

Comment:
`.querySelectorAll(...)[0]` can be simplified.
```suggestion
		if (document.querySelector(".react-jinke-music-player-pause-icon"))
```